packages/arb-token-bridge-ui/src/components/MainContent/ArbitrumStats.tsx

```diff
@@ -0,0 +1,104 @@
+import { utils } from 'ethers'
+import { useArbQueryParams } from 'src/hooks/useArbQueryParams'
+import { useGasPrice } from 'token-bridge-sdk'
+import { useArbStats } from '../../hooks/useArbStats'
```

Comment:
better add a linebreak above this line to separate the two import groups

Reply:
If we want to have sorted import, then we should have prettier/lint for it, it should not be done PR by PR

Reply:
agree, last time we discussed that we should get it done when all the PRs are merged
i don't foresee it to happen soon though